### DISCUSSION TITLE
Customize Renovate's status check strings

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#23141] [Suggest an Idea]
### What would you like Renovate to be able to do?

I would like to think if it's the right time to let users customize Renovate's status check strings too

_Originally posted by @rarkins in https://github.com/renovatebot/renovate/issues/21376#issuecomment-1504727770_

### If you have any ideas on how this should be implemented, please tell us here.

I'll let the maintainers figure out the requirements and implementation details. 😉 

### Is this a feature you are interested in implementing yourself?

No

**Comment by HonkingGoose** [comment]
@rarkins is this still a thing you're interested in adding? If yes, we should list the requirements and code changes. That way we can copy/paste the good stuff into a actionable Issue.

**Comment by rarkins** [comment]
Created feature request issue #23530
